src/main/java/seedu/address/logic/commands/people/PeoplePaidCommand.java

```diff
@@ -0,0 +1,135 @@
+package seedu.address.logic.commands.people;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliPrefix.PEOPLE_COMMAND_TYPE;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.tag.Tag;
+import seedu.address.model.transaction.Amount;
+import seedu.address.model.transaction.Debt;
+import seedu.address.model.transaction.Loan;
+import seedu.address.model.transaction.TransactionList;
+
+/**
+ * Records that the person, who owe the user money has paid the user back.
+ */
+public class PeoplePaidCommand extends Command {
```

Comment:
I feel like the commandWord should be a verb in simple tense: pay. Like all other commands (add, delete, lend) etc

Reply:
But we are recording it after the person paid the user back, shouldn't it be a past tense?

Reply:
should i change mine to returned?

Reply:
Can, actually should I change mine to received (from the user perspective)? Since paid and returned sounds similar

Reply:
yea i think should